blender-cli/src/contract.rs

```diff
@@ -227,6 +227,69 @@ impl Blender {
             _ => panic!("Expected {:?} to be a Tuple", &value),
         }
     }
+
+    pub fn register_new_token(
+        &self,
+        connection: &SignedConnection,
+        token_id: u16,
+        token_address: AccountId,
+    ) -> Result<()> {
+        let subscription = subscribe_events(connection)?;
+        let (cancel_tx, cancel_rx) = channel();
+        let (signal_tx, signal_rx) = channel::<()>();
+
+        let contract_ptr = self.contract.clone();
+
+        let token_address_copy = token_address.clone();
+
+        thread::spawn(move || {
+            listen_contract_events(
+                subscription,
+                &[&contract_ptr],
+                Some(cancel_rx),
+                |event_or_error| {
+                    println!("{:?}", event_or_error);
+                    if let Ok(ContractEvent { ident, data, .. }) = event_or_error {
+                        if Some(String::from("TokenRegistered")) == ident {
```

Comment:
Do we have such event? If so, then it would be good to have `token_id` and `contract_address` inside it (for identification here)